src/frontend/src/pages/LandingPage.tsx

```diff
@@ -27,7 +27,7 @@ function LandingPage() {
             <section className="what-is-a-pia">
                 <h2>What is a Privacy Impact Assessment</h2>
                 <p>A PIA is a step-by-step review process to make sure you protect the personal information you collect, use or disclose in your project. Doing a PIA can help protect privacy and build public trust by being clear about what information government is collecting, who has access to it, and where and how it’s stored.</p> 
-                <a href='/'>Learn about the current PIA process</a>
+                <a href='https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/privacy/privacy-impact-assessmentS'>Learn about the current PIA process</a>
```

Comment:
Nit: I usually like seeing smaller lines - and that should be taken care by Linter anyways. Now I am wondering if Husky is even working? @binggaofw

Reply:
Looks like we only have Linting for Backend in package.json. That too doesn't seem to work. @binggaofw - request you please give it a look when you have time next?

Reply:
i only config the husky for backend